main.go

```diff
@@ -17,23 +17,48 @@ limitations under the License.
 package main
 
 import (
+	"log"
+	"os"
+
 	appcatv1 "github.com/vshn/appcat-apiserver/apis/appcat/v1"
 	"github.com/vshn/appcat-apiserver/apiserver/appcat"
 	"github.com/vshn/appcat-apiserver/apiserver/vshn/postgres"
 	"k8s.io/klog"
 	"sigs.k8s.io/apiserver-runtime/pkg/builder"
 )
 
+var ()
+
 func main() {
-	err := builder.APIServer.
-		// +kubebuilder:scaffold:resource-register
-		WithResourceAndHandler(&appcatv1.AppCat{}, appcat.New()).
-		WithResourceAndHandler(&appcatv1.VSHNPostgresBackup{}, postgres.New()).
-		WithoutEtcd().
+
+	var appcatEnabled bool
+	var vshnEnabled bool
+
+	if os.Getenv("APPCAT_ENABLED") == "1" {
```

Comment:
```suggestion
	if os.Getenv("APPCAT_HANDLER_ENABLED") == "1" {
```

I would change a bit the name to be more explicit.